SalesforceCommerceCloud/pwa-kit
# Remove the PersistentCache functionality

# Description

The Managed Runtime team has removed the PersistentCache functionality on our platform. These changes are to remove the PersistentCache functionality in the PWA Kit before removing the entire PersistentCache feature in a future release.

# Types of Changes

Removing the underlying PersistentCache functionality.


# Changes

- Remove PersistentCache functionality
- Update tests

# How to Test-Drive This PR

- Add the cache code (below) to the template-retail-react-app `ssr.js`
- Deploy these changes to a storefront and hit the cache endpoints. They should all work with the `get` always not found
  - `/get?k=key`
  - `/put?k=key&v=value`
  - `/delete?k=key` 

### Code
```
app.get('/get', (req, res) => {
    const k = req.query.k
    app.applicationCache
        .get({key: k})
        .then((e) => {
            res.send(e)
        })
        .catch((e) => {
            console.error(e), res.send(e)
        })
})
app.get('/put', (req, res) => {
    const k = req.query.k
    const v = req.query.v
    app.applicationCache
        .put({key: k, data: v})
        .then((e) => {
            console.log(e), res.send(`PUT ${k}: ${v}`)
        })
        .catch((e) => {
            console.error(e), res.send(e)
        })
})
app.get('/delete', (req, res) => {
    const k = req.query.k
    app.applicationCache
        .delete({key: k})
        .then((e) => {
            console.log(e), res.send(`DELETE ${k}`)
        })
        .catch((e) => {
            console.error(e), res.send(e)
        })
})
```

## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
